main.go

```diff
@@ -8,6 +8,11 @@ import (
 	log "github.com/Sirupsen/logrus"
 	"github.com/rancherio/go-machine-service/events"
 	"github.com/rancherio/go-machine-service/handlers"
+	_ "github.com/rancherio/go-machine-service/handlers/providers/amazonec2"
```

Comment:
It wasn't needed before because the handlers for each provider(azure, amazon etc.) were in the handlers package itself. So, when i imported handlers earlier, all of these were also loaded. Now, they are in their own separate packages, and I need to load them individually.